darwin/exporter/formats/nifti.py

```diff
@@ -185,7 +190,18 @@ def process_metadata(metadata):
                 volume_dims = volume_dims[1:]
         else:
             volume_dims = None
-    return volume_dims, pixdim, affine
+    return volume_dims, pixdim, affine, original_affine
+
+
+def process_affine(affine):
+    if isinstance(affine, str):
+        affine = np.squeeze(np.array([eval(l) for l in affine.split("\n")]))
```

Comment:
Can we limit the available locals/globals dict to `eval` to prevent a injection vectors?